docs/configuration/tools/router_check.rst

```diff
@@ -0,0 +1,109 @@
+.. _config_tools_router_check_tool:
+
+Route table check tool
+======================
+
+**NOTE: The following configuration is for the router table check tool only and is not part of the Envoy binary.
+The route table check tool is used for testing purposes only.**
+
+Input for the route table check tool. The route table check tool checks if the route returned
```

Comment:
Do you intend for `Input for the route table check tool` to be a header? Otherwise it doesn't make too much sense the way the sentence is structured.